index.html

```diff
@@ -13,6 +13,9 @@
       <meta property="og:site_name" content="Creative Commons">
       <title>CC Search</title>
       <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/foundation/6.4.3/css/foundation.min.css">
+      <link rel="icon" href="https://d15omoko64skxi.cloudfront.net/wp-content/uploads/2016/05/cc-site-icon-150x150.png" sizes="32x32">
+      <link rel="icon" href="https://d15omoko64skxi.cloudfront.net/wp-content/uploads/2016/05/cc-site-icon-300x300.png" sizes="192x192">
```

Comment:
I think you should add a `type="image/png"` to both of these

Reply:
Hi @brenoferreira ,

It has been 7 days since your requested review. I wonder; Do you still work on it @geekayush ?
As we are going to ship `CC Search 1.0` and we don't have a `favicon` yet. 

Please give me the green light to offer some help here!

Reply:
@elharony I was busy with my tests. I have pushed the changes.